### ISSUE TITLE
hello.js prints undefined?

### ISSUE THREAD

**Issue by kokizzu** [just-js/just#13]
```
just init hello
cd hello
just build 
time ./hello
```
output: `undefined`

version used: 0.0.22

**Comment by kokizzu** [comment]
`just eval "just.print(just.memoryUsage().rss)"` works tho
```
9187328
```

but after creating new project, it shows this error:
```
build blabla 0.0.22 (just.js) g++ -c -fno-exceptions -ffunction-sections -fdata-sections  -DJUST_VERSION='"0.0.22"' -std=c++17 -DV8_COMPRESS_POINTERS -I. -I./deps/v8/include -O3 -march=native -mtune=native -Wpedantic -Wall -Wextra -flto -Wno-unused-parameter just.cc
g++ -c -fno-exceptions -ffunction-sections -fdata-sections  -std=c++17 -DV8_COMPRESS_POINTERS -I. -I./deps/v8/include -O3 -march=native -mtune=native -Wpedantic -Wall -Wextra -flto -Wno-unused-parameter main.cc
g++ -s -static -flto -pthread -m64 -Wl,--start-group deps/v8/libv8_monolith.a main.o just.o builtins.o modules/sys/sys.o modules/fs/fs.o modules/net/net.o modules/vm/vm.o modules/epoll/epoll.o -Wl,--end-group -Wl,--gc-sections  -ldl -lrt -o blabla
/usr/bin/ld: /tmp/blabla.pU5vnP.ltrans0.ltrans.o: in function `_sub_I_65535_0.8055':
<artificial>:(.text.startup+0xb7): undefined reference to `_binary_blabla_js_end'
/usr/bin/ld: <artificial>:(.text.startup+0xbe): undefined reference to `_binary_blabla_js_start'
/usr/bin/ld: /tmp/blabla.pU5vnP.ltrans2.ltrans.o: in function `main':
<artificial>:(.text.startup+0x1e9): undefined reference to `_binary_blabla_js_start'
/usr/bin/ld: <artificial>:(.text.startup+0x1f7): undefined reference to `_binary_blabla_js_end'
collect2: error: ld returned 1 exit status
make: *** [Makefile:87: main-static] Error 1
failed in 4.6 sec
```

**Comment by billywhizz** [comment]
yes. it looks like a couple of bugs here. there's a lot of things that need to be fixed and will have a roadmap for a stable release soon. looks like the code here is out of date: https://github.com/just-js/just/blob/main/lib/build.js#L311.

and if you do 
```
just build --clean
```

it should resolve the issue with the missing symbols. i'll have a fix for this out shortly.

**Comment by billywhizz** [comment]
closing this as these issues should be resolved in upcoming 0.1.2 release
